File: IntroJava/src/SimpleIfDemo.java
/*
Listing 3.2 SimpleIfDemo.java
 */

import java.util.Scanner;

public class SimpleIfDemo
{
    public static void main(String[] args)
    {
        Scanner input = new Scanner(System.in);
        System.out.print("Enter a number: ");
        int number = input.nextInt();

        if (number % 2 == 0)
            System.out.println("Hi even!");
        else
            System.out.println("Hi odd!");
    }
}

File: IntroPython/SimpleIfDemo.py
# LISTING 4.2 SimpleIfDemo.py

import random

# number = eval(input("Enter an integer: "))
number = random.randint(0, 9)

if number % 5 == 0:
    print("Hi Five.")

if number % 2 == 0:
    print("Hi even.")
else:
    print("Hi", number)
